test/espnet2/asr/encoder/test_transformer_encoder.py

```diff
@@ -15,8 +15,6 @@ def test_Encoder_forward_backward(input_layer, positionwise_layer_type):
     )
     if input_layer == "embed":
         x = torch.randint(0, 10, [2, 10])
-    elif input_layer is None:
```

Comment:
Why did you remove this test?

Reply:
There is no need for a special test for input_layer none. It is handled due to the changes I made in `transformer_encoder.py`